### ISSUE TITLE
Getting the status of previously run job in its very next run !

### ISSUE THREAD

**Issue by ApurvaPatil** [node-schedule/node-schedule#294]
Its just a query..but may become an issue. Can we get the status of previously triggered job ( whether it is still running or completed ) in its very next run ?

In the job, I am actually running a asyncronous process. So, it may happen that a set of previously run asynchronous process might not have completed their execution. So, need this requirement. Whether it is already implemented ? If yes, it would be really helpful if I get an explaination for the same. If not, it would be nice to have kind of feature !

**Comment by techtolentino** [comment]
+1

**Comment by santigimeno** [comment]
@ApurvaPatil No, at the moment there's no way. Contributions are certainly welcomed. In way, the issue is basically the same as in thehttps://github.com/node-schedule/node-schedule/issues/298

**Comment by iwozzy** [comment]
You might be able to do it like this:

```javascript
var schedule = require('node-schedule');
var isJobRunning = false;

var job = schedule.scheduleJob('*/5 * * * * *', function () {
  if (!isJobRunning) {
    console.log('Running');
    // logic here
    
    setTimeout(function() {
      // job finished
      isfetchImagesJobRunning = false
    }, 10000);
  } else {
    // do nothing
    console.log('Job already running');
  }
});
```
